### ISSUE TITLE
Source code sharing request

### ISSUE THREAD

**Issue by SUOXIAOXI** [fuqichen1998/SequentialVotingDet#3]
Hello!
Thank you for your nice works!
I read your article《Deformer: Dynamic Fusion Transformer for Robust Hand Pose Estimation》while reviewing the literature and will the code of this article be updated?

**Comment by fuqichen1998** [comment]
Yes, the code should be released soon, and the link will be updated here.
